bundles/EcommerceFrameworkBundle/IndexService/Tool/IndexUpdater.php

```diff
@@ -161,6 +173,8 @@ public static function processUpdateIndexQueue($tenants = null, $maxRounds = nul
             $tenants = [$tenants];
         }
 
+        self::startSession();
```

Comment:
Are duplicate `startSession()`s required?

Reply:
Duplicated starts aren't necessary, but it's necessary to ensure the session is ready to use in every possible entry point which could trigger an output before starting the session.
A duplication of session starts is explicitly covered in the helper method which only starts a session if the code runs in CLI and there no session.

Reply:
Oh, I was confused by Github's review interface, didn't see that they where in two different methods. It looked like they where just on different sides of an foreach. Sorry! :D

Reply:
No worries :)